tests/EvmAcceptanceTests/test/Erc20.js

```diff
@@ -45,14 +45,17 @@ describe("ERC20 functionality", function () {
     });
     // TODO: Add event related test(s), e.g. for Transfer event
 
-    // FIXME: In ZIL-4879
-    xit("Should not be possible to move more than available tokens to some address [@transactional]", async function () {
+    it("Should not be possible to move more than available tokens to some address [@transactional]", async function () {
       const [owner, receiver] = await ethers.getSigners();
       const prevBalance = await contract.balanceOf(owner.address);
 
       // Move one coin more than available coins.
-      await contract.transfer(receiver.address, prevBalance + 1);
-
+      // Workaround below since for some reason chai assertions don't handle exceptions properly,
+      // e.g. expect(cond).to.throw() doesn't work here
+      try {
+        await contract.transfer(receiver.address, prevBalance + 1);
```

Comment:
if it reverts the transaction you can use [revert assertions](https://hardhat.org/hardhat-chai-matchers/docs/overview#reverts)

Reply:
Tried that, it actually doesn't revert but throws an exception during gas estimation.